src/pyre_reduction.cc

```diff
@@ -14,57 +15,51 @@ namespace recycle {
 
 Reduct::Reduct() {}
```

Comment:
Default constructor should construct a valid instance.
```suggestion
Reduct::Reduct() {
  set_coeff();
  current(0);
  lithium(0);
  volume(0);
  Rtime(0);
}
```